src/poliastro/tests/test_plotting.py

```diff
@@ -69,6 +69,6 @@ def test_color():
 
 
 def test_plot_solar_system():
-    op = plot_solar_system()
-    assert len(op._orbits) == 8
-    assert isinstance(op, OrbitPlotter)
+        assert len(plot_solar_system(True).orbits) == 8
+        assert len(plot_solar_system(False).orbits) == 4
```

Comment:
To parametrize the `outer` parameter, use `pytest.mark.parametrize`: https://docs.pytest.org/en/stable/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions